example/ios/Podfile

```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-# platform :ios, '10.0'
+# platform :ios, '9.0'
```

Comment:
why are we downgrading?

Reply:
It's not downgrading since it's commented out. But basically this Podfile was regenerated by flutter, and it generates the comment with 9.0 version by default